### PR TITLE
Show all patients and single patient

### PR DESCRIPTION
**Pull request by Angie254**

### CODE REVIEW

**Comment by Infinical** [comment]
peek 2019-02-01 17-36

File: lib/nailinda/user/user.ex [hunk 1]
```diff
@@ -9,4 +9,11 @@ defmodule Nailinda.User do
     	|>Patient.changeset(attrs)
     	|> Repo.insert()
     end
+    def get_patient do
```
Comment (GettyOrawo) [inline lib/nailinda/user/user.ex]: this function should say what it does

**Comment by Infinical** [comment]
peek 2019-02-01 17-54

**Review by okothkongo — COMMENTED** [review]
you can limit PR to issue related to keep it clean and make easy to review, for instance index route is not related to this pr

**Comment by Infinical** [comment]
peek 2019-02-01 18-01

**Review by GettyOrawo — APPROVED** [review]

**Review by LindaKadz — COMMENTED** [review]
Please do the refactor, piping of a raw value to a function, before we merge it. Thanks!

**Pull request merged**